Gemfile

```diff
@@ -4,9 +4,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in ombu_labs-auth.gemspec.
 gemspec
 
-gem "pg"
+gem 'sqlite3', '~> 1.5', '>= 1.5.3'
 
 gem "sprockets-rails"
 
 # Start debugger with binding.b [https://github.com/ruby/debug]
-# gem "debug", ">= 1.0.0"
+gem "debug", ">= 1.0.0"
```

Comment:
@arielj Please either remove or uncomment this line. I think it might be useful for debugging bugs, so you might want to keep it in there.

Reply:
sorry, I'm not sure what you mean, this is uncommenting this line, you mean to re-add the comment?

Reply:
@arielj oh got it, I thought you were removing it. 😄